docs/get-started.md

```diff
@@ -0,0 +1,167 @@
+# Instrument a Go application or services automatically
+
+You can instrument a Go executable using OpenTelemetry without having
+to write additional code. All you need to do is configure a few environment
+variables and run the instrumentation with elevated privileges.
+
+The following example shows how to instrument a simple Go application
+automatically on a Linux host, through Docker, and using Kubernetes.
+
+## Prerequisites
+
+To instrument an application automatically, you need the following:
+
+- Binary file or Docker image of OpenTelemetry Go instrumentation
+- Go 1.17 and higher in your system
```

Comment:
I'm not sure you need Go installed if you already have a binary or docker image of the auto-instrumentation, right?

If you want to build the binary it will require 1.18, not 1.17, though.

Reply:
@pellared We're not offering a binary of the instrumentation yet thought, right?

Reply:
Right, we are not offering a binary for download yet. Only a docker image.

Maybe we want to say you will need the docker image or the source code and at least Go 1.18 to compile a binary?